src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
-            </NextLink>
-          </ListItem>
-          <ListItem disableGutters>
-            <NextLink href={`/organize/${orgId}/projects`} passHref>
-              <IconButton
-                aria-label="Projects"
-                className={classes.roundButton}
-                color={key.startsWith('/projects') ? 'primary' : 'secondary'}
-                data-test="calendar-button"
-                size="large"
+  const [open, setOpen] = useState(false);
+  const toggleDrawer = () => {
+    setOpen(!open);
+    setExpanded(!expanded);
+  };
+
+  const model = useModel((env) => new OrganizationsDataModel(env));
+
+  const menuItemsMap = [
+    { icon: <Map />, name: 'areas' },
+    { icon: <Home />, name: 'home' },
+    { icon: <Explore />, name: 'journeys' },
+    { icon: <People />, name: 'people' },
+    { icon: <Event />, name: 'projects' },
+  ] as const;
+
+  const OrganizeSideBar = (
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
+      <Drawer
+        onFocus={() => void 0}
+        onMouseLeave={() => {
+          setHover(false);
+        }}
+        onMouseOver={() => {
+          setHover(true);
+        }}
+        open={open}
+        sx={{
+          alignItems: open ? 'flex-start' : 'center',
+        }}
+        variant="permanent"
+      >
+        <DrawerHeader>
+          <IconButton onClick={toggleDrawer}>
+            {!open && hover && !expanded && (
+              <KeyboardDoubleArrowRightIcon
+                onClick={toggleDrawer}
+                sx={{ margin: 1 }}
+              />
+            )}
+            {!open && !hover && !expanded && (
+              <NextLink href="/organize" passHref>
+                <Avatar
+                  alt="icon"
+                  src={`/api/orgs/${orgId}/avatar`}
+                  sx={{ alignSelf: 'center' }}
+                />
+              </NextLink>
+            )}
+            {expanded && (
+              <ZUIFuture future={model.getOrganization(orgId)}>
+                {(data) => {
+                  return (
+                    <Box alignSelf="center">
+                      <Box display="flex" justifyContent="start">
+                        {hover ? (
+                          <ChevronLeftIcon
+                            onClick={handleDrawerToggle}
+                            sx={{ alignSelf: 'center', margin: 2 }}
+                          />
+                        ) : (
+                          <Avatar
+                            alt="icon"
+                            src={`/api/orgs/${orgId}/avatar`}
+                            sx={{ alignSelf: 'center', marginLeft: 2 }}
+                          />
+                        )}
+                        <Typography gutterBottom m={2} variant="h6">
+                          {data.title}
+                        </Typography>
+                      </Box>
+                    </Box>
+                  );
+                }}
+              </ZUIFuture>
+            )}
```

Comment:
I don't think all of this should be inside the `IconButton`. Only the chevron should be in a button.

Reply:
Agree!